symengine/cwrapper.cpp

```diff
@@ -160,7 +160,7 @@ TypeID basic_get_type(const basic s)
     return static_cast<TypeID>(s->m->get_type_code());
 }
 
-void symbol_set(basic s, char *c)
+void symbol_set(basic s, const char *c)
```

Comment:
Now that I think about it, `const char *` is enough. It doesn't matter if the pointer is assigned to or not as long as the memory pointed to is not changed.